src/noStyles.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+var ReactTelephoneInput = require('./ReactTelephoneInput');
+module.exports = ReactTelephoneInput;
```

Comment:
Applications can require this file to get the telephone input without the styles.